### DISCUSSION TITLE
Just small query

### DISCUSSION THREAD

**Discussion by ghost** [finos/a11y-theme-builder#849] [General]
Hi! Do we need to be a previous contributor to this project in order to apply for the LFX mentorship? Thankyou!

**Comment by ghost** [comment]
@aaronreed708 @evangk6 Can you clarify this?

**Reply by aaronreed708** [reply]
No, you certainly do not need to be a previous contributor to Theme Builder to be considered for the mentorship.

**Reply by ghost** [reply]
ok .Thank you.
